integration/flink/src/main/java/io/openlineage/flink/visitor/DatasetFactory.java

```diff
@@ -35,17 +35,24 @@ public OpenLineage.Builder<OpenLineage.OutputDataset> datasetBuilder(
     };
   }
 
+  public OpenLineage.DatasetFacetsBuilder getDatasetFacetsBuilder(String name, String namespace) {
```

Comment:
Would be nice to add "java common" library similar to what we have in Python as we do common gradle build. 

Well, at some point... not at this PR though.